package.json

```diff
@@ -15,7 +15,8 @@
       "types/broccoli-concat",
       "types/ember-cli-htmlbars",
       "tests/scenarios",
-      "tests/app-template"
+      "tests/app-template",
+      "tests/addon-template"
```

Comment:
Why it is in `tests/` instead of `test-packages`? Not a blocker or anything, just curious...

Reply:
As we refactor each test package into a scenario I am moving them over to keep them separate. I prefer the final state of just having 'tests' and that test-packages will be fully gone once all of them are converted.